wetterdienst/provider/dwd/mosmix/api.py

```diff
@@ -518,8 +514,6 @@ def _all(self) -> pd.DataFrame:
 
         df = df[(df.iloc[:, 0] != "=====") & (df.iloc[:, 0] != "TABLE") & (df.iloc[:, 0] != "clu")]
 
-        df = df.iloc[:, [2, 3, 4, 5, 6, 7]]
-
```

Comment:
Do we need this line, @gutzbenj? It croaked after adjusting the colspecs, so I've just removed it ;].

Reply:
So there would be a column missing with the new format?